vm-aws-yaml/Pulumi.yaml

```
@@ -0,0 +1,124 @@
+name: vm-aws-yaml
+description: A YAML program to create a virtual machine on AWS
```

To allow these names to be set at `pulumi new`-time, we use these expressions:

```suggestion
name: ${PROJECT}
description: ${DESCRIPTION}
```